### PR TITLE
Rename update to upgrade in update.use-cli.php

### LINKED ISSUE #39028: [QA] login page says 'update' needed, when an 'upgrade' is needed
Seen while upgrading from 10.7.0 to 10.8.0 RC1

image

This message is inconsistent with the occ command required. the occ command reads 'upgrade', the message reads 'update'.

Expected behaviour: same wording. Use 'upgrade' in the message.

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue
- Fixes https://github.com/owncloud/core/issues/39028

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by JanAckermann** [comment]
@phil-davis do we need changelogs if we simply optimize texts ?

File: core/templates/update.use-cli.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <div class="update" data-productname="<?php p($_['productName']) ?>" data-version="<?php p($_['version']) ?>">
 	<div class="updateOverview">
-		<h2 class="title"><?php p($l->t('Update needed')) ?></h2>
+		<h2 class="title"><?php p($l->t('Upgrade needed')) ?></h2>
 		<div class="infogroup">
 			<?php if ($_['tooBig']) {
 	p($l->t('It looks like your instance may host many files and/or users. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).'));
```
Comment (phil-davis) [inline core/templates/update.use-cli.php:6]: I guess this line is OK - "please use the command line updater (occ upgrade)" 
"command line upgrader" sounds strange to me in English.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Do we write changelogs for this sort of stuff?

File: core/templates/update.use-cli.php (hunk 1 above)
Comment (JanAckermann) [reply core/templates/update.use-cli.php:6]: please use the command line upgrade command instead?

Comment (phil-davis) [reply core/templates/update.use-cli.php:6]:
```suggestion
	p($l->t('It looks like your instance may host many files and/or users. To ensure a smooth upgrade process, please use the command line upgrade command (occ upgrade).'));
```

@jnweiger what do you think should be written here?

**Comment by JanAckermann** [comment]
@phil-davis IHMO I would suggest not do a changelog item for every single translation update, because this could possibly extend the changelog (which should help the user to see what is new in a version) in an unwanted detailed way.
Just my 2 cents

**Review by phil-davis — APPROVED** [review]

**Comment by mmattel** [comment]
Rebased to restart CI

**Comment by phil-davis** [comment]
@micbar what is the rule for changelogs in this situation?
Only some text on the UI  and potential "error" messages has changed. There is not "real behavior" change.

**Comment by phil-davis** [comment]
@pmaier1  what is the rule for changelogs in this situation?
Only some text on the UI  and potential "error" messages has changed. There is not "real behavior" change.

**Comment by pmaier1** [comment]
> @pmaier1 what is the rule for changelogs in this situation?

Any change should be reflected in the changelog. Can you add an entry, please?

**Comment by phil-davis** [comment]
Changelog added, rebased an squashed,

**Review by phil-davis — APPROVED** [review]

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIProxySmoke-3-2-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/31856/171/1

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/31856/171/16
`Fatal error: Call to undefined method GuzzleHttp\Exception\ConnectException::getResponse() (Behat\Testwork\Call\Exception\FatalThrowableError)`

I have seen that happen before but rarely - restarting CI

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIAddUsers-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/31858/130/1

**Review by JammingBen — APPROVED** [review]

**Comment by mmattel** [comment]
Info: Transifex has been updated just now to translate the changes made to `de` and `de_DE`

**Pull request merged**